### ISSUE TITLE
testing application data set and steps

### ISSUE THREAD

**Issue by cambro** [UW-Deepdive-Infrastructure/app-template#23]
this needs to be added. currently, running the test application gives this:
`cursor.execute("SELECT * FROM stringed_instruments_sentences_nlp352;")
psycopg2.ProgrammingError: relation "stringed_instruments_sentences_nlp352" does not exist
LINE 1: SELECT * FROM stringed_instruments_sentences_nlp352;`

The reason is because the tables defined by the steps to import the testing set are not correct. I would make the testing set data = 1 data set for all steps involved here. So, let's make this data from "Getting Started":

`curl -o example_input.zip https://geodeepdive.org/dev_subsets/example_input.zip
unzip -j example_input.zip -d ./input
rm example_input.zip`

Be the same data that is used by the example application, and perhaps get rid of weird table names etc. in the postgres database.

**Comment by cambro** [comment]
@iross When fed-up with ingest documentation and processing overhaul, do a "from scratch" app-template follow through as if you were newbie. This helps find errors.

**Comment by iross** [comment]
Part of the issue is that the script we use to read the example_input into tables was looking for config instead of config.yml, which has been fixed. 

The other issue is that getting the example_input into the database is accomplished via a `make local_setup`, which gets a little buried in the 'Getting started' guide. While it'd be nice to have one copy/paste block to run the example app, there's always going to be the step of modifying the credentials to match the local postgres setup. But I do think there's value in having a more streamlined quickstart/how to run the example app wiki page (and I think we've had a placeholder for it at https://github.com/UW-Deepdive-Infrastructure/app-template/wiki/Running-the-example-application)

**Comment by cambro** [comment]
Yes, that's the idea. This placeholder page is what caught my attention and led me to try and run the example app. As you say, it would really help if this were streamlined for users (config. for db. connection is a separate issue that IS in fact addressed in the documentation).
